spec/requests/api/v1/resources/update_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe "update resource", type: :request do
+  let(:identifier) { 'update-test' }
+
+  let(:valid_update_attributes) {
+    {
+      width: 2000
+    }
+  }
+
+  let(:invalid_update_attributes) {
+    {
+      width: 'string'
+    }
+  }
+
+  describe "PATCH /resources/:id" do
+    let(:patch_url) { "/api/v1/resources/#{identifier}.json" }
+    it "returns a success response for a valid attribute update" do
+      FactoryBot.create(:resource, identifier: identifier)
+      patch patch_url, params: { resource: valid_update_attributes }
+      expect(response).to have_http_status(:success)
+    end
+
+    it "returns a 400 unprocessable entity response for an invalid attribute update" do
+      FactoryBot.create(:resource, identifier: identifier)
+      patch patch_url, params: { resource: invalid_update_attributes }
+      expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
I thought we talked about just returning :bad_request for errors like this. I just want to use consistent error messaging across our apps. I can understand if there's a reason to separate :bad_request and :unprocessable_entity errors for some reason.

Reply:
a tangential small point: unprocessable entity is 422, not 400.

Reply:
@cgalarza I copied what you had in Hyacinth 3, so that's why I went for :unprocessable_entity rather than :bad_request, but I just realized that my local copy of Hyacinth 3 branched off of some much earlier code, so I'm not up to date.  Will change to :bad_request.  Thanks!